swap/swap.go

```diff
@@ -149,13 +150,17 @@ func newSwapInstance(stateStore state.Store, owner *Owner, backend contract.Back
 // - connects to the blockchain backend;
 // - verifies that we have not connected SWAP before on a different blockchain backend;
 // - starts the chequebook; creates the swap instance
-func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Params, chequebookAddressFlag common.Address, initialDepositAmountFlag uint64, factoryAddress common.Address) (swap *Swap, err error) {
+func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Params, chequebookAddressFlag common.Address, skipDepositFlag bool, depositAmountFlag uint64, factoryAddress common.Address) (swap *Swap, err error) {
 	// swap log for auditing purposes
 	swapLog = newSwapLogger(params.LogPath, params.OverlayAddr)
 	// verify that backendURL is not empty
 	if backendURL == "" {
 		return nil, errors.New("no backend URL given")
 	}
+	// verify that depositAmountFlag and skipDeposit are not conflicting
+	if depositAmountFlag > 0 && skipDepositFlag {
```

Comment:
I think these comparison make much more sense with the new flag name, doesn't it?

Reply:
I does ;)